frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/SelectedTaskDrawerContent.tsx

```diff
@@ -21,7 +20,14 @@ const SelectedTaskDrawerContent: React.FC<SelectedTaskDrawerContentProps> = ({ t
   }
 
   return (
-    <DrawerPanelContent data-testid="task-drawer" style={{ height: '100%', overflowY: 'auto' }}>
+    // TODO: Revisit below approach, either to further to look into what caused the content to not render or
+    // to restructure the code
+    <div
+      className="pf-v6-c-drawer__panel"
+      data-testid="task-drawer"
+      // TODO: look into removing this inline style; PF Drawers should handle height/scrolling by default
+      style={{ height: '100%', overflowY: 'auto' }}
+    >
```

Comment:
@Gkrumbach07 this is something @jenny-s51 has talked to me about before I approved the merge. This is a hacky workaround for now, we need to make sure this is tracked via a Jira ticket on the Dashboard.